force-app/main/default/lwc/lmsSubscriberWebComponent/lmsSubscriberWebComponent.js

```diff
@@ -4,12 +4,7 @@ import { ShowToastEvent } from 'lightning/platformShowToastEvent';
 import { reduceErrors } from 'c/ldsUtils';
 
 // Import message service features required for subscribing and the message channel
-import {
-    subscribe,
-    unsubscribe,
-    APPLICATION_SCOPE,
-    MessageContext
-} from 'lightning/messageService';
+import { subscribe, MessageContext } from 'lightning/messageService';
 import recordSelected from '@salesforce/messageChannel/Record_Selected__c';
```

Comment:
Use uppercase for the channel name because it's a constant (unlike the other imports). BTW this is consistent with doc.

Reply:
done

Reply:
Also updated this in the lmsPublisherWebComponent.js module.